snap/snapcraft.yaml

```diff
@@ -311,3 +312,13 @@ parts:
       for res in "${resources[@]}"; do
           rm -rf "${CRAFT_PART_INSTALL}/${res}"
       done
+
+    override-prime: |
+      # Copy wrappers for bin commands to fix upgrades: https://github.com/canonical/opensearch-operator/issues/537
+      craftctl default
+
+      mv "${CRAFT_PRIME}/usr/share/opensearch/bin/opensearch-plugin" "${CRAFT_PRIME}/usr/share/opensearch/bin/opensearch-plugin.orig"
+      mv "${CRAFT_PRIME}/usr/share/opensearch/bin/opensearch-keystore" "${CRAFT_PRIME}/usr/share/opensearch/bin/opensearch-keystore.orig"
+
+      cp "${CRAFT_STAGE}/opt/opensearch/plugin-wrapper.sh" "${CRAFT_PRIME}/usr/share/opensearch/bin/opensearch-plugin"
+      cp "${CRAFT_STAGE}/opt/opensearch/keystore-wrapper.sh" "${CRAFT_PRIME}/usr/share/opensearch/bin/opensearch-keystore"
```

Comment:
I don't think we should override the built-in bin files of opensearch. Any reason to do it?

Reply:
thank you!